Translated/Dialog/Dighal/ssq_012222.csv

```diff
@@ -0,0 +1,3 @@
+name_7417#0,"""Dighal"""
+ssq_012222_0010#0,"""Alright, looks like you didn't have any trouble being able to gather them."""
```

Comment:
"didn't have any trouble Gathering them."?

Reply:
yeah that was bugging me too, sec

Reply:
fixt